Makefile

```diff
@@ -75,16 +83,6 @@ format:
 test: mimiker.elf
 	./run_tests.py
 
-# Detecting whether initrd.cpio requires rebuilding is tricky, because even if
```

Comment:
What was the rationale behind moving `initrd` stuff to subdirectory? Isn't that an unnecessary change?

Reply:
Since the source files are already separated into different directories (i.e. `mips`, `sys`, etc.), I thought I'd follow this pattern. I also didn't want to create intermediate files (`initrd.o`, `initrd.cpio`) in the root directory of the project.

Reply:
Hmm... `initrd` directory stores only `Makefile` at the moment and it's unlikely it will contain anything else in foreseeable future. I'll revert that change. We should discuss such decisions next time.